tycho-its/src/test/java/org/eclipse/tycho/test/target/TargetPlatformLocationsTest.java

```diff
@@ -126,6 +133,31 @@ public void testMavenLocationAutogeneratedFeature() throws Exception {
 		Verifier verifier = getVerifier("target.maven.autofeature", false, true);
 		verifier.executeGoal("verify");
 		verifier.verifyErrorFreeLog();
+
+		Path targetPlatformRepository = Path.of(verifier.getBasedir(),
+				"test.product/target/targetPlatformRepository/content.xml");
+		try (var stream = Files.newInputStream(targetPlatformRepository)) {
+			Xpp3Dom dom = Xpp3DomBuilder.build(stream, StandardCharsets.UTF_8.displayName());
+			Xpp3Dom[] children = dom.getChild("units").getChildren("unit");
+			Optional<Xpp3Dom> sourceFeature = Stream.of(children)
+					.filter(it -> "org.apache.commons.io.feature.feature.group".equals(it.getAttribute("id")))
```

Comment:
```suggestion
					.filter(it -> "org.apache.commons.io.source..feature.feature.group".equals(it.getAttribute("id")))
```
Probably this one is actually wanted (but just guessing)?

Reply:
Yep, that's the failing test. 😖